.github/workflows/upmerge.yaml

```diff
@@ -79,4 +79,4 @@ jobs:
         if: env.NO_CHANGES != 'true'
         env:
           GITHUB_TOKEN: ${{ secrets.GH_RAD_CI_BOT_PAT}}
-        run: gh pr create --title "Upmerge to edge" --body "Upmerge to edge (kicked off by @${{ github.triggering_actor }})" --base edge --head $BRANCH_NAME --signoff
+        run: gh pr create --title "Upmerge to edge" --body "Upmerge to edge (kicked off by @${{ github.triggering_actor }})" --base edge --head $BRANCH_NAME
```

Comment:
We require commits to be signed off for PRs so wondering if this will remove the auto-signoff. It looks like the flag is meant to be used with `git commit` so we should update the workflow in edge to include that here: https://github.com/radius-project/docs/blob/d425e1d78551bcfecb6c50583c2e8849e311918f/.github/workflows/upmerge.yaml#L67

Reply:
I don't think this needs to block the release but we should make sure to update the workflow after

Reply:
agreed. BTW for reference the samples repo does not sign off commits today when we upmerge.

Reply:
opened issues:

https://github.com/radius-project/docs/issues/1270
https://github.com/radius-project/samples/issues/1884